news/editions/edition4.md

```diff
@@ -0,0 +1,81 @@
+
+## [Spinnaker Operator Demo](https://www.youtube.com/watch?v=VojwvcxZF3k)
+Take a look at the Spinnaker Operator, a Kubernetes operator for Spinnaker that makes it easier to install, deploy, and upgrade any version of Spinnaker using standard Kubernetes application workflows and tools, like kubectl, helm, or kustomize.
+
+## [Spinnaker idea: reusable run job stages via script runner containers](https://medium.com/@tomas_lin/spinnaker-idea-reusable-run-job-stages-via-script-runner-containers-ff5fd95ec056)
+Combine the deployment ease of the run script stage with the security advantages of a containerized stage by building general runner containers that run mix-and-match script workflows from S3. Enjoy the flexibility and security of a container runtime, and reduce time spend building containers.
+
+## [A Guide On The Installation Of Spinnaker in Kubernetes Clusters](https://www.magalix.com/blog/a-guide-on-the-installation-of-spinnaker-in-your-production-kubernetes-cluster)
+With this guide, use Halyard to install Spinnaker in a Kubernetes cluster, such as one you create in Google Kubernetes Engine. Configure an IAM role, storage bucket, and Halyard, and deploy a production-grade instance of Spinnaker.
+
+## [Spinnaker projects participating in Google Summer of Code](https://cd.foundation/blog/2020/05/18/9-cd-foundation-projects-are-participating-in-this-years-google-summer-of-code/)
+This year, we have joined GSoC to bring more student developers into the Spinnaker community. We'll nurture two Spinnaker projects: "Drone CI type for Spinnaker pipeline stage," from Victor Odusanya mentored by Armory engineer [Cameron Motevasselani](https://www.youtube.com/watch?v=HtkXeC8a38Y); and “Continuous Delivery, Continuous Deployments with Spinnaker,” from Moki Daniel mentored by Armory engineer [Fernando Freire](https://www.armory.io/blog/identifying-risk-when-executing-your-kubernetes-migration/).
+
+## [The Continuous Delivery Foundation, interoperability, and open standards](https://www.infoq.com/podcasts/continuous-delivery-foundation/)
+Tracy Miranda, the new Governing Board chair of the CDF, discusses the foundation's purpose and interoperability goals on the InfoQ podcast. Read more about Tracy's vision of standardized metadata in continuous delivery pipelines, and how Spinnaker's advanced deployment strategies fit in.
+
+## [Spinnaker community builder elected as chair of CD Foundation Outreach Committee](https://cd.foundation/announcement/2020/04/24/new-chair-of-cd-foundation-outreach-committee-elected/)
+We are excited to announce that [Rosalind Benoit](https://blog.spinnaker.io/building-open-source-community-culture-online-part-1-2f77272f5442) was elected the CD Foundation Outreach Committee Chairperson. The Outreach Committee manages marketing and outreach for CDF projects on behalf of the Governing Board. Rosalind’s election recognizes her contributions to CDF community efforts, and will bring Spinnaker outreach into focus.
```

Comment:
```suggestion
We are excited to announce that [Rosalind Benoit](https://blog.spinnaker.io/building-open-source-community-culture-online-part-1-2f77272f5442) was elected chairperson of the CD Foundation Outreach Committee. The Outreach Committee manages marketing and outreach for CDF projects on behalf of the Governing Board. Rosalind’s election recognizes her contributions to CDF community efforts, and will bring Spinnaker outreach into focus.
```

Reply:
for `and will bring Spinnaker outreach into focus`
* No comma before `and`
* I suggest using present tense instead of future, i.e. `and brings Spinnaker outreach into focus.` But also, `into focus` or `into the spotlight`? Focus implies it was blurry and unclear before.